File: Basics_with_JavaScript/34_First_Steps_in_Coding/More_Exersises/06._Fishland.js
function vladi(input){

    skumriaPrice = Number(input[0]);
    cacaPrice = Number(input[1]);
    kgPalamud = Number(input[2]);
    kgSafrid = Number(input[3]);
    kgMidi = Number(input[4]);
    
    palamudPrice = skumriaPrice * 1.6
    safridPrice = cacaPrice * 1.8
    midiPrice = 7.5

    bill = (palamudPrice * kgPalamud + safridPrice * kgSafrid + midiPrice * kgMidi).toFixed(2)
    console.log(bill);
    
    }
   
    vladi([6.90, 4.20, 1.5, 2.5, 1]);
    vladi([5.55, 3.57, 4.3, 3.6, 7]);
    vladi([7.79, 5.35, 9.3, 0, 0]);

File: Basics_with_Python/PythonBasicCourse/1_First_Steps_In_Coding/4_First_Steps_in_Coding_Exercises/more_exercises/06._Fishland.py
price_skumr = float(input())
price_caca = float(input())
kg_palamud = float(input())
kg_safrid = float(input())
kg_midi = int(input())

price_palamud = 1.6 * price_skumr
price_safrid = 1.8 * price_caca
price_midi = 7.50
total = price_midi * kg_midi + price_safrid * kg_safrid + price_palamud * kg_palamud
print(f"{total:.2f}")
